internal/gps/vcs_repo.go

```diff
@@ -80,27 +82,54 @@ func newVcsLocalErrorOr(msg string, err error, out string) error {
 }
 
 func (r *gitRepo) get(ctx context.Context) error {
-	out, err := runFromCwd(ctx, expensiveCmdTimeout, "git", "clone", "--recursive", "-v", "--progress", r.Remote(), r.LocalPath())
-	if err != nil {
-		return newVcsRemoteErrorOr("unable to get repository", err, string(out))
+	cmd := exec.CommandContext(
+		ctx,
+		"git",
+		"clone",
+		"--recursive",
+		"-v",
+		"--progress",
+		r.Remote(),
+		r.LocalPath(),
+	)
+	if out, err := cmd.CombinedOutput(); err != nil {
+		return newVcsRemoteErrorOr(
+			"unable to get repository",
+			errors.Wrapf(err, "command failed: %v", cmd.Args),
+			string(out),
+		)
 	}
 
 	return nil
 }
 
 func (r *gitRepo) fetch(ctx context.Context) error {
-	// Perform a fetch to make sure everything is up to date.
-	out, err := runFromRepoDir(ctx, r, expensiveCmdTimeout, "git", "fetch", "--tags", "--prune", r.RemoteLocation)
-	if err != nil {
-		return newVcsRemoteErrorOr("unable to update repository", err, string(out))
+	cmd := exec.CommandContext(
+		ctx,
+		"git",
+		"fetch",
+		"--tags",
+		"--prune",
+		r.RemoteLocation,
+	)
+	cmd.Dir = r.LocalPath()
+	if out, err := cmd.CombinedOutput(); err != nil {
+		return newVcsRemoteErrorOr(
+			"unable to update repository",
+			errors.Wrapf(err, "command failed: %v", cmd.Args),
```

Comment:
these errors are going to end up being `errors.Wrap`'d higher up in the call stack - i'm not sure we gain much by doing it again here.

Reply:
I believe these wraps are preserving the wrap that was happening inside of `runFromRepoDir` via `combinedOutput`

Reply:
ahh, i see, that's right, you just added those. OK

(aside - man i wish we had consistent strategies for error handling in various subsystems)